### DISCUSSION TITLE
Support encrypted group message

### DISCUSSION THREAD

**Discussion by vwumumu** [MixinNetwork/mixin#149] [Q&A]
There is an [issue](https://github.com/MixinNetwork/bot-api-nodejs-client/issues/224) about "Support encrypted group message" has been created for "bot-api-nodejs-client" on Jun 13.
May I have an update if there is an ETA for this function?

**Comment by cedricfung** [comment]
Nobody is working on it yet, but could be soon. The Golang SDK does have this feature already. https://github.com/MixinNetwork/bot-api-go-client

**Reply by vwumumu** [reply]
Thanks for your quick response to this. I am not familiar with Golang, and my bot build with Node. Anyway, appreciate it will be available soon.
